Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'github-markdown' # only for dashboard controller
 # all other gems
 gem 'devise'
 gem 'decent_exposure'
+gem 'interactor-rails'
```

Comment:
@timurvafin, I'm a bit concerned about using `interactor-rails` instead of `interactor`:

* `interactor-rails` is not updated in time with releasing new versions of `interactor` gem
* `interactor-rails` provides only dumb scaffold for interactors/organizers, but _without_ any specs

What do you think about using `interactor` instead?